### PR TITLE
Introduce a welcome screen

### LINKED ISSUE #2271: Show welcome message instead of Wizard for new users
I would suggest that we don't throw new users into the Wizard, but instead show a welcome screen:

I think we may have discussed it before a long time ago. I base it on the reaction I have seen when introducing xonsh to new users. Most people find the xonsh exiting when I tell about it, and they want to try it fast. Then when it is finally installed the wizard becomes a 'turn-off' when all they want is try it out fast.

So what about a welcome screen instead when no rc files are present. 

```
             Welcome to the xonsh shell (v0.57)

   ~ Python-powered, cross-platform, Unix-gazing shell ~    

xonfig tutorial -> Launch a tutorial in browser http://xon.sh/tutorial.html
xonfig wizard   -> Configure the shell and dispell this welcome message
                   (or create ./xonshrc file)
user@home ~/ $ 
```

We should of course have most of the tag-lines in the welcome message as well.

### PR DESCRIPTION
**Pull request by melund**
This adds a welcome screen which is shown instead of throwing new users directly into the Wizard. 

image

I have no idea how it looks on Linux or OSX. All ideas and improvements are welcome :) 

closes #2271

### CODE REVIEW

**Comment by melund** [comment]
I should automatically adapt to the terminal width. Do we have an easy way of getting that?

**Comment by scopatz** [comment]
Thanks so much @melund! The code looks good to me. Just waiting on CI.

**Comment by laerus** [comment]
this looks awesome, will try it tomorrow on debian, does `os.get_terminal_size` helps?

File: xonsh/xonfig.py [hunk 1]
```diff
@@ -512,3 +521,88 @@ def xonfig_main(args=None):
...
+    '',
+    ('{{INTENSE_RED}}~{{NO_COLOR}} {tagline} {{INTENSE_RED}}~{{NO_COLOR}}', '^', ' '),
+    '',
+    ('{{INTENSE_BLACK}}', '<', '-'),
+    '{{GREEN}}xonfig{{NO_COLOR}} tutorial    {{INTENSE_WHITE}}->    Launch the tutorial in '
+    'the browser{{NO_COLOR}}',
+    '{{GREEN}}xonfig{{NO_COLOR}} wizard      {{INTENSE_WHITE}}->    Run the configuration '
+    'wizard and claim your shell {{NO_COLOR}}',
+    '{{INTENSE_BLACK}}(Note: Run the Wizard or create a {{RED}}~/.xonshrc{{INTENSE_BLACK}} file '
+    'to suppress this meassage)',
```
Comment (gforsyth) [inline xonsh/xonfig.py]: *message

**Comment by gforsyth** [comment]
Looks awesome, @melund!

**Pull request merged**